content/en/docs/reference/access-authn-authz/validating-admission-policy.md

```diff
@@ -373,7 +373,7 @@ When an API request is validated with this admission policy, the resulting audit
 In this example the annotation will only be included if the `spec.replicas` of the Deployment is more than
 50, otherwise the CEL expression evaluates to null and the annotation will not be included.
 
-Note that audit annotation keys are prefixed by the name of the `ValidatingAdmissionWebhook` and a `/`. If
+Note that audit annotation keys are prefixed by the name of the `ValidatingAdmissionPolicy` and a `/`. If
```

Comment:
Maybe this:
```suggestion
Audit annotation keys are prefixed by the name of the admission webook or ValidatingAdmissionPolicy, with a `/` separator. If
```

Reply:
Validating/Mutating admission webhooks can add audit annotations using the `AdmissionReview.Response.AuditAnnotations` field, which are prefixed with the admission webhook name. However, I believe the sentence in this documentation refers specifically to audit annotations added by validating admission policies, rather than a general statement about audit annotations added by all admission controllers.

Reply:
Ah, good point.

Try this:
```suggestion
Audit annotation keys are prefixed by the name of the ValidatingAdmissionPolicy and a `/`. If
```